tox.ini

```diff
@@ -1,7 +0,0 @@
-[tox]
-envlist = py37,py38,py39,py310,py311
```

Comment:
This was testing against a bunch of different Python versions. Is that now done somewhere else?

Edit: looks like [github actions is configured to do it](https://github.com/dimagi/jsonobject/blob/eb4d2ddd9fcbe145c32ef9d3638d7c8a4f54a38f/.github/workflows/tests.yml#L18), so should be safe to remove here, assuming this is not used elsewhere.